samples/outbox/sqltransport-nhpersistence-ef/SqlServer_3/Receiver/ReceiverDataContext.cs

```diff
@@ -1,30 +1,39 @@
-using System.Data.Entity;
-using System.Data;
+using Microsoft.EntityFrameworkCore;
 using System.Data.Common;
 
 public class ReceiverDataContext :
     DbContext
 {
     #region EntityFramework
-    public ReceiverDataContext()
-        : base("NServiceBus/Persistence")
+    public ReceiverDataContext(DbConnection connection)
     {
+        this.connection = connection;
     }
-    public ReceiverDataContext(IDbConnection connection)
-        : base((DbConnection) connection, false)
+    public ReceiverDataContext(string connectionString)
     {
+        this.connectionString = connectionString;
     }
-    #endregion
-
-    public DbSet<Order> Orders { get; set; }
 
-    protected override void OnModelCreating(DbModelBuilder modelBuilder)
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
     {
-        base.OnModelCreating(modelBuilder);
+        if (IsConnectionProvided(connection))
+        {
+            optionsBuilder.UseSqlServer(connection);
+        }
+        else
+        {
+            optionsBuilder.UseSqlServer(connectionString);
+        }
```

Comment:
we dont need the `optionsBuilder.ConfigureWarnings(wcb => wcb.Ignore(RelationalEventId.AmbientTransactionWarning))` here?